### PR TITLE
fix(catalog link): copy should catalog, not course catalog

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Updating "Course Catalog" text for the catalog header item to shortened "Catalog" 

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: EGG-773
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@9.2.10-alpha.86d709.0<br/>@codecademy/styleguide@25.1.11-alpha.86d709.0<br/></pre>

File: packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const AppHeaderLink: React.FC<AppHeaderLinkProps> = ({
     <AppHeaderLinkButtonOuter
       data-testid={item.dataTestId}
       href={item.href}
-      onClick={(event) => action(event, item)}
+      onClick={(event: React.MouseEvent) => action(event, item)}
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: Err, is this necessary? Is it not inferable as a `MouseEvent`?

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60246d8de7d119680c5eeb74--gamut-preview.netlify.app>https://60246d8de7d119680c5eeb74--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60246d8de7d119680c5eeb74>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx (hunk 1 above)
Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: i thought it would be but my vscode did not like it

Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: getting no implicit any errors

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: hmm.. HMM....

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: Maybe your local repo needs a `build-all`? If I `yarn lerna clean` then `yarn` I can get it into this state:

A TypeScript error importing from Gamut at the to of the file, with a noImplicitAny error on event

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@9.3.1-alpha.c96f5f.0<br/>@codecademy/styleguide@25.2.1-alpha.c96f5f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://602470bdee9f6d6c654972b9--gamut-preview.netlify.app>https://602470bdee9f6d6c654972b9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/602470bdee9f6d6c654972b9>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
✨

File: packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx (hunk 1 above)
Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: this is exactly the sate I was in!

Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: after adding the type to event on line 31, i no longer see that squiggly - but I sitll see the one at the top 'cannot find gamut'

Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: (even after running the yarn commands & rebuilding)

Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/AppHeaderElements/AppHeaderLink/index.tsx:31]: oh you know what! i was using a linked version of React from the monolith; removing that and then running the yarn commands & rebuilding fixed it!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@9.3.1-alpha.a0de7a.0<br/>@codecademy/styleguide@25.2.1-alpha.a0de7a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60247f015db2b175d94b2ecb--gamut-preview.netlify.app>https://60247f015db2b175d94b2ecb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60247f015db2b175d94b2ecb>Deploy Logs</a></p>

**Pull request merged**